test/FunctionalTests/Tests/DataServices/ddbasics/TestHttpClientProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.OData.Client;

```

Comment:
Is this not the same class as
https://github.com/OData/odata.net/pull/2883/files#diff-f4748932855371bed900c751c5215fdb57f98380b5cff81c55af54288b58d4c5
Why not re-use it ?

Reply:
@habbes Is there a "TestCommon" project that you can move the test classes that you have redefined in different test projects?

Reply:
The `TestCommon` project doesn't have references to OData core libraries and I'm not too fond of changing the dependencies in this pipeline solution until we port/fix it for ODL 8. Otherwise I'd have moved it there.

Also, I don't think it's a bad thing to have this class duplicated. It's small and doesn't do much. And if it changed in different ways in each of the 3 projects, that would be fine since each project might have unique scenarios it tests for.